src/Analysis/Ast/Impl/Types/PythonType.cs

```diff
@@ -24,7 +24,7 @@ namespace Microsoft.Python.Analysis.Types {
     [DebuggerDisplay("{Name}")]
     internal class PythonType : LocatedMember, IPythonType {//, IEquatable<IPythonType> {
         private readonly object _lock = new object();
-        private readonly string _name;
+        protected readonly string _name;
         private Dictionary<string, IMember> _members;
```

Comment:
If it is protected, make it a read-only property. Just .NET style thing.

Reply:
Ok